pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -510,6 +510,9 @@ type VirDomain interface {
 	AbortJob() error
 	Free() error
 	CoreDumpWithFormat(to string, format libvirt.DomainCoreDumpFormat, flags libvirt.DomainCoreDumpFlags) error
+	PinVcpuFlags(vcpu uint, cpuMap []bool, flags libvirt.DomainModificationImpact) error
+	PinEmulator(cpumap []bool, flags libvirt.DomainModificationImpact) error
+	SetVcpusFlags(vcpu uint, flags libvirt.DomainVcpuFlags) error
```

Comment:
Right... This is being done in a different way because the domain is running now.

Until now, we didn't handle pinning but simply generated a correct domain XML and delegated the handling to libvirt. This time, the domain is already running and we need to pin the additional threads.